proto/import_sstpb.proto

```diff
@@ -143,6 +143,12 @@ message DownloadRequest {
     backup.StorageBackend storage_backend = 14;
 }
 
+// For now it is just used for distinguishing the error of the request with the error 
+// of gRPC, add more concrete types if it is necessary later.
+message Error {
+    string message = 1;
```

Comment:
Could you include errorpb.Error here?

Reply:
I think it is not necessary, currently it is just used to return the error of `DownloadSST`.